commons/build.gradle

```diff
@@ -55,6 +55,11 @@ android {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
 
+    debugImplementation project(':core')
+    releaseImplementation("org.hisp.dhis:android-core:${libraries.dhis2sdk}") {
+        exclude group: "org.hisp.dhis", module: "core-rules"
+    }
+
```

Comment:
If we are adding the sdk dependency here, maybe it would a good idea to make available to the app module via the "releaseApi" and "debugApi" rather than having it duplicated in the App module. Maybe we can discuss it with the team.

Reply:
Yes, this could be a good idea :)